lib/libzfs/libzfs_sendrecv.c

```diff
@@ -2968,6 +2969,17 @@ zfs_receive_one(libzfs_handle_t *hdl, int infd, const char *tosnap,
 	zc.zc_begin_record = drr_noswap->drr_u.drr_begin;
 	zc.zc_cookie = infd;
 	zc.zc_guid = flags->force;
+
+	if (zfs_dataset_exists(hdl, zc.zc_value, ZFS_TYPE_SNAPSHOT) &&
+	    flags->skip) {
+		(void) printf("skipping snapshot %s - %s already exists\n",
+				drrb->drr_toname, zc.zc_value);
```

Comment:
Normally I'd say never to use `printf()` in a library function like this.  Unfortunately, it's already being done in this function so let's just make sure it's wrapped with an `if (flags->verbose)` conditional like the other cases.